packages/server-wallet/src/wallet/__test__/fixtures/participants.ts

```diff
@@ -1,24 +1,24 @@
-import {Participant, makeDestination} from '@statechannels/wallet-core';
+import {Participant, makeDestination, makeAddress} from '@statechannels/wallet-core';
 
 import * as wallets from './signing-wallets';
 import {fixture} from './utils';
 
 const _alice: Participant = {
-  signingAddress: wallets.alice().address,
+  signingAddress: makeAddress(wallets.alice().address),
```

Comment:
Curious why this is necessary since `SigningWallet.address` is of type `Address`?

Reply:
Good call, I did this when I was hitting a bizarre stress test error. Fixed in 6c1b1a05d